src/components/ForgotPassword/ForgotPassword.jsx

```diff
@@ -1,29 +1,155 @@
 import React, { useState } from 'react';
-import { Button } from 'react-bootstrap';
-import styled from 'styled-components';
-
-const SubmitButton = styled(Button)`
-    background-color: #6593D6;
-    color: white;
-    padding: 10px;
-    border-radius: 10px
-    border: none;
-`;
-
-export default () => {
-  const [email, setEmail] = useState('');
-
-  const checkEmail = () => {
-    if (email === '') {
-      window.alert('Enter an email');
+import { Auth } from 'aws-amplify';
+import { Link } from 'react-router-dom';
+import {
+  FormGroup,
+  FormControl,
+  FormLabel,
+} from 'react-bootstrap';
+
+export default function ResetPassword() {
+  const [code, setCode] = useState('');
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+
+  const [codeSent, setCodeSent] = useState(false);
+  const [confirmed, setConfirmed] = useState(false);
+  const [isConfirming, setIsConfirming] = useState(false);
+  const [isSendingCode, setIsSendingCode] = useState(false);
+
+  function validateCodeForm() {
+    return username.length > 0;
+  }
+
+  function validateResetForm() {
```

Comment:
General comment, but try to use more arrow functions. While most of these functions are simple, arrow functions are more commonly used throughout our project and are easier to read.